fileutil/file_writer.go

```diff
@@ -129,7 +134,7 @@ func (bw *BufferedFileWriter) trySyncFileRange() error {
 		syncOffset -= syncOffset % bytesAlign
 		syncSize := syncOffset - bw.lastSyncOffset
 		if syncOffset > 0 && syncSize >= bw.bytesPerSync {
-			err = SyncFileRange(bw.f, bw.lastSyncOffset, syncSize, true)
+			err = SyncFileRange(bw.f, bw.lastSyncOffset, syncSize, false)
```

Comment:
Can we remove the bool argument?

Reply:
I think this is a util function, it's better to provide this argument to let caller control its behavior. Maybe we will use `async sync_file_range` again letter.

Reply:
Then how about we use another function `SyncFileRangeAsync` for later use?
A bool argument is hard to read.

Reply:
Got it.